modules/getting-started-web-console-load-data-output.adoc

```diff
@@ -1,39 +1,41 @@
 // Module included in the following assemblies:
 //
-// * getting-started/openshift-web-console.adoc
+// * tutorials/dev-app-web-console.adoc
 
 :_mod-docs-content-type: PROCEDURE
 [id="getting-started-web-console-load-data-output_{context}"]
+= Loading data into the database
 
-= Loading data and displaying the national parks map
-
-You deployed the `parksmap` and `nationalparks` applications and then deployed the `mongodb-nationalparks` database. However, no data has been loaded _into_ the database.
-Before loading the data, add the proper labels to the `mongodb-nationalparks` and `nationalparks` deployment.
+After you have deployed the `mongodb-nationalparks` database, you can load the national park location information into the database.
 
 .Prerequisites
 
-* You are logged in to the {product-title} web console.
-* You are in the *Developer* perspective.
-* You have a deployed image.
+* You are in the *Developer* perspective of the web console.
```

Comment:
we disabled `Developer` perspective by default in OCP 4.19, `Workloads -> Topology` is moved to `Administrator` perspective though

Reply:
Good catch, I forgot to remove it from this one.